LICENSE.md

```diff
@@ -0,0 +1,22 @@
+# MIT License
+
+Copyright (c) 2014 Hua Qiu
```

Comment:
Probably a typo and it should be 2024 :-)

Reply:
Thanks for the headup, @a-maze-d !